packages/fxa-graphql-api/src/config.ts

```diff
@@ -58,6 +58,11 @@ const conf = convict({
     default: 'http://localhost:7000',
     env: 'CUSTOMS_SERVER_URL',
   },
+  settingsUrl: {
+    doc: 'Settings server url',
+    default: 'http://localhost:3030',
+    env: 'SETTINGS_SERVER_URL',
```

Comment:
Thanks for the reminder!

Reply:
This was more so about trying to land it in the dot release earlier this week FWIW and having the deploy doc updated in time for SRE. I know you don't need a reminder about this otherwise 😂